crates/sqlexec/src/extension_codec.rs

```diff
@@ -18,6 +18,8 @@ pub struct GlareDBExtensionCodec<'a> {
     table_providers: Option<&'a HashMap<Uuid, Arc<dyn TableProvider>>>,
 }
 
+// This tracks all of our extensions so that we can ensure an exhaustive match on anywhere that uses the extension
```

Comment:
The codec? Or is there something missing here?

Reply:
whoops, i originally had `ExtensionType` in this file, but moved it to `extension`. i guess i forgot to move the comment.